backend/api/serializers/CreditTrade.py

```diff
@@ -404,8 +404,22 @@ def get_comments(self, obj):
 
         return serializer.data
 
-    def get_reviewed(self, obj):
-        serializer = CreditTradeHistoryReviewedSerializer(obj.reviewed)
+    def get_history(self, obj):
```

Comment:
Could we create a test to cover the new endpoint?

Reply:
Apparently, there was a test for this already and I just didn't update it. It should now be updated.